src/views/Feed/index.d.ts

```diff
@@ -128,18 +136,20 @@ interface FeedExtraProps {
   className?: string;
 
   /** Shorthand for primary content. */
-  content?: any;
+  content?: React.ReactNode;
 
   /** An event can contain additional information like a set of images. */
-  images?: any;
+  images?: boolean | Array<any>;
```

Comment:
We set `React.ReactNode` when we deal with `contentShorthand`, but in this case we have:
```js
PropTypes.oneOfType([
  PropTypes.bool,
  customPropTypes.collectionShorthand,
]),
```

Which represents array of `itemShorthand`, we mark it as `any` for now.

I want to add new TS types for `customPropTypes` in future.